src/config/firebase-config.ts

```diff
@@ -0,0 +1,59 @@
+import { initializeApp } from "firebase/app";
+import {
+  getFirestore,
+  collection,
+  getDocs,
+  addDoc,
+  deleteDoc,
+  serverTimestamp,
+  doc,
+  orderBy,
+  limit,
+  onSnapshot,
+  query,
+} from "firebase/firestore";
+import {
+  getAuth,
+  createUserWithEmailAndPassword,
+  updateProfile,
+  onAuthStateChanged,
+  signInWithEmailAndPassword,
+  signOut,
+} from "firebase/auth";
+
+const firebaseConfig = {
```

Comment:
I see that the .env file already contains these config values. You could take these values from there instead as it's more secure.

Reply:
Resolved it. I had an error initially with the .env file, but I saw now that I have typed "REACT__APP" with two '_' instead of one.